fastlane/Fastfile

```diff
@@ -409,60 +534,6 @@ platform :ios do
     end
   end
 
-  #####################################################################################
-  # build_and_upload_beta
-  # -----------------------------------------------------------------------------------
-  # This lane builds the app and uploads it for distribution
-  # -----------------------------------------------------------------------------------
-  # Usage:
-  # bundle exec fastlane build_and_upload_beta [skip_confirm:<skip confirm>]
-  #
-  # Example:
-  # bundle exec fastlane build_and_upload_beta
-  # bundle exec fastlane build_and_upload_beta skip_confirm:true
-  #####################################################################################
-  desc 'Builds and uploads for distribution'
-  lane :build_and_upload_beta do |options|
```

Comment:
After searching the release scenario, Buildkite scripts, and Buildkite pipelines, I did not find any instances of this lane being used.